boards/openmoteb/board.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ * Copyright (C) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_openmoteb
```

Comment:
can we name it `boards_openmote_b` and the board directory should be `openmote-b` to be consistent with the rest